frontend/src/DevoFeedback.css

```
@@ -0,0 +1,334 @@
+
+.feedback-content {
+  flex: 1;
+}
+
+.feedback-text {
+  color: var(--color-text-primary);
+  font-size: var(--font-size-base);
+  line-height: 1.6;
+  margin: 0;
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.4.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)